scripts/us_eia/opendata/process/common.py

```diff
@@ -53,27 +54,41 @@ def _parse_date(d):
             # Monthly
             return yr + '-' + m_or_q
 
+    if len(d) == 8:
+        # PET has weekly https://www.eia.gov/opendata/qb.php?sdid=PET.WCESTUS1.W
+        yr = d[:4]
+        m = d[4:2]
+        dt = d[6:2]
+        return yr + '-' + m + '-' + dt
+
     return None
 
 
 def _eia_dcid(raw_sv):
     return 'dcid:eia/' + raw_sv.lower()
 
 
+def _enumify(in_str):
+    return in_str.title().replace(' ', '')
```

Comment:
do we need to ensure camel case as well?

Reply:
title() approx does that?

https://stackoverflow.com/questions/3728655/titlecasing-a-string-with-exceptions

Reply:
oh, i missed that there was a call to "title" -- sg